bin/ci/script.sh

```diff
@@ -34,3 +32,9 @@ if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" &&
     kill -0 $CHROMEDRIVER_PID
   )
 fi
+
+if [[ "$ORCA_JOB" ]]; then
+  eval "orca ci:run $ORCA_JOB script $ORCA_SUT_NAME"
+fi
+
+echo "completed"
```

Comment:
@TravisCarden : I am observing a weird behaviour, though this line has no significance at all, if I remove this line the allowed failure feature is not working and tests are failing. Do you know why?

Reply:
That _is_ weird; I _don't_ know why. Would it behave the same way with lines 36-38 back up at the beginning of the file where they were before? Because presumably this was working before, and that's the only significant difference I see.

Reply:
If we put 36-38 where it was before, it would work absolutely fine. But I wanted to run the `Nightwatch` tests before we run `orca ci:run $ORCA_JOB script $ORCA_SUT_NAME` so that we can capture Nightwatch failures in Domo. But if I re-arrange the lines then the `allowed failure` feature is not working not sure why. Then I have to add the line `echo "completed"` or anything else to make it work. 

In short, it seems we cannot put the `orca ci:run $ORCA_JOB script $ORCA_SUT_NAME` part at the bottom of the file.

Reply:
Well, then I'd say choose which is more important to you (behavior that your fully understand vs. behavior that does what you want) and leave inline comments. If you're dissatisfied with that, take a personal note--or if it's important enough to you, create a Jira issue--and come back to it when it offers more value than competing priorities. `¯\_(ツ)_/¯`